src/phantom.js

```diff
@@ -75,9 +76,15 @@ export default class Phantom {
         this.process.on('error', error => {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
             logger.error(error);
+            this.kill('Process got an error');
```

Comment:
The _rejectAllCommands method rejects with a new error, so when printed you get Error: errmsg